tests/decode/test5_strange/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ if (TEST_INDETERMINATE_LENGTH_ARRAYS)
   target_compile_definitions(app PUBLIC TEST_INDETERMINATE_LENGTH_ARRAYS)
 endif()
 
+if (VERBOSE)
+  target_compile_definitions(app PUBLIC ZCBOR_VERBOSE)
+endif()
```

Comment:
Would it be better if all of these `CMakeLists.txt` included a common file `test.cmake` which did this? Maybe we should add the `CANONICAL` test there as well.

That is, if it is sufficient to add the definitions to the `zehpyr_interface` target (which I think is linked with all of the test targets.